src/api/IBMi.ts

```diff
@@ -180,16 +176,33 @@ export default class IBMi {
         const checkShellText = `This should be the only text!`;
         const checkShellResult = await this.sendCommand({
           command: `echo "${checkShellText}"`,
-          directory: `.` // Will use whatever the user profile default is
+          directory: `/` // Will use root as the working directory for this command
```

Comment:
The `directory` has no effect and should be removed - the SSH server is changing directory to the non-existing user home directory before running the command and will return `Could not chdir to home directory /home/xxxxxxx: No such file or directory`

```suggestion
          command: `echo "${checkShellText}"`
        });
        if (checkShellResult.code === 0 && checkShellResult.stdout.split(`\n`)[0] !== checkShellText) {
```